### PR TITLE
prometheus: upgrade prometheus components

### PR DESCRIPTION
**Pull request by pgier**
prometheus 2.2.1 -> 2.3.1
alertmanager 0.14.0 -> 0.15.0
node_exporter 0.15.2 -> 0.16.0

### CODE REVIEW

File: roles/openshift_prometheus/defaults/main.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ openshift_prometheus_alertmanager_hostname: alertmanager-{{openshift_prometheus_
 l_openshift_prometheus_image_version: "{{ openshift_prometheus_image_version | default('v2.2.1') }}"
 l_openshift_prometheus_alertmanager_image_version: "{{ openshift_prometheus_alertmanager_image_version | default('v0.14.0') }}"
 l_openshift_prometheus_alertbuffer_image_version: "{{ openshift_prometheus_alertbuffer_image_version | default('v0.0.2') }}"
-l_openshift_prometheus_node_exporter_image_version: "{{ openshift_prometheus_node_exporter_image_version | default('v0.15.2') }}"
+l_openshift_prometheus_node_exporter_image_version: "{{ openshift_prometheus_node_exporter_image_version | default('v0.16.0') }}"
```
Comment (vrutkovs) [inline roles/openshift_prometheus/defaults/main.yaml]: This can safely be removed - its not used anywhere

**Comment by pgier** [comment]
bot, retest this please

**Comment by pgier** [comment]
bot, retest this please

**Comment by pgier** [comment]
bot, retest this please

**Pull request merged**